pkg/subctl/cmd/gather/types.go

```diff
@@ -29,4 +30,5 @@ type Info struct {
 	DynClient   dynamic.Interface
 	ClusterName string
 	DirName     string
+	Submariner  submariner.Submariner
```

Comment:
Since this isn't use by all components, I think it should just be stored and passed locally.

Reply:
Ack. Was in two minds regarding this. Will make it local for now and add if later more components need this.